worker/config/environment.py

```diff
@@ -27,6 +27,8 @@ class Settings(BaseSettings):
     FINISHED_FILE_FORMAT: str = ".txt"
     ZIP_NOT_DEFINED_USER_INPUT: str = "auto"
 
+    LOG_LEVEL: str = "DEBUG"
```

Comment:
The default log level should still be `INFO`.
```suggestion
    LOG_LEVEL: str = "INFO"
```